src/libraries/System.Globalization/tests/System/Globalization/RegionInfoTests.cs

```diff
@@ -63,8 +64,9 @@ public void CurrentRegion_Icu()
             }
         }
 
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNlsGlobalization))]
-        public void TestCurrentRegion_Nls()
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void TestCurrentRegion_Windows()
```

Comment:
Nit: Why `CurrentRegion` vs `TestCurrentRegion` in these two test names?

Reply:
I don't know 😄 -- will update to be consistent.